includes/admin/class-sensei-extensions.php

```diff
@@ -212,6 +212,31 @@ private function get_has_update_count() {
 		);
 	}
 
+	/**
+	 * Get installed Sensei plugins.
+	 *
+	 * @param bool $only_woo Only include WooCommerce.com extensions.
+	 *
+	 * @return array
+	 */
+	public function get_installed_plugins( $only_woo = false ) {
+		$extensions = $this->get_extensions( 'plugin' );
+
+		return array_filter(
+			$extensions,
+			function( $extension ) use ( $only_woo ) {
+				if (
+					! $extension->installed_version
```

Comment:
This one throws a PHP error notice when not set. 

```suggestion
					empty( $extension->installed_version )
```